bittensor/synapse.py

```diff
@@ -689,11 +637,6 @@ def parse_headers_to_inputs(cls, headers: dict) -> dict:
         inputs_dict["header_size"] = headers.get("header_size", None)
         inputs_dict["total_size"] = headers.get("total_size", None)
         inputs_dict["computed_body_hash"] = headers.get("computed_body_hash", None)
-        inputs_dict["hash_fields"] = json.loads(
```

Comment:
(same question as the previous one) does the axon need to know the other parsed hash_fields other then the required_hash_field?